#global testerror "Err0341"
func contains(what: u8) => return Ut8.contains(toSlice(), what)